src/bank.rs

```diff
@@ -178,7 +179,7 @@ impl Bank {
             loader: solana_native_loader::id(),
         };
         self.accounts
-            .store_slow(&system_program::id(), &system_program_account);
+            .store_slow(true, &system_program::id(), &system_program_account);
```

Comment:
if you store(tokens=0) with purge=true onto a fork with a parent, it exposes the parent's state, right?